tests/factories/item_factories.py

```diff
@@ -15,6 +15,7 @@ class ItemFactory(SQLAlchemyModelFactory):
     summary = 'Summary of the feed item.'
     text = 'Text of the feed item.'
     feed = factory.SubFactory(FeedFactory)
+    totalVotes = 0
```

Comment:
Might be worth leaving in case we want to change it. Every other column of the model is represented here, it'd be weird to leave one out.